CHANGELOG.rst

```diff
@@ -55,6 +55,8 @@ Improvements
 ------------
 - When NLU training data is dumped as Markdown file the intents are not longer ordered
   alphabetically, but in the original order of given training data
+- Added a random-seed parameter on the rasa data split nlu command allowing reproducible
+  nlu data splits.
```

Comment:
We recently changed the way we add changelog entries. Please, have a look here: https://github.com/RasaHQ/rasa/tree/master/changelog